app/components/settings/billing-page/support-section.hbs

```diff
@@ -0,0 +1,18 @@
+<Settings::FormSection @title="Support" @description="How to get help with billing" data-test-support-section>
+  <a
+    href="mailto:hello@codecrafters.io?subject=Billing help (account: {{@user.username}})"
+    class="inline-block rounded"
+    data-test-support-contact-button
+  >
+    <PrimaryButton @size="small">
+      Get help
+    </PrimaryButton>
+  </a>
+  <div class="text-gray-500 dark:text-gray-400 text-xs mt-3">
```

Comment:
Are you sure this should be `text-xs` and not `text-sm`? Seems like we're using `text-sm` for the other sections. 

Also, @Arpan-206 I'm pretty sure the original design preview was using `prose` classes here - we usually don't have to specify any of these styles for text. Can you double check please?